templates/logging-sidecar-configmap.yaml

```diff
@@ -32,60 +32,12 @@ data:
           component: "${COMPONENT:--}"
           workspace: "${WORKSPACE:--}"
           release: "${RELEASE:--}"
-
-      filter_common_logs:
-        type: filter
-        inputs:
-          - transform_syslog
-        condition:
-          type: "vrl"
-          source: '!includes(["worker","scheduler"], .component)'
-
-      filter_scheduler_logs:
-        type: filter
-        inputs:
-          - transform_syslog
-        condition:
-          type: "vrl"
-          source: 'includes(["scheduler"], .component)'
-
-      filter_worker_logs:
-        type: filter
-        inputs:
-          - transform_syslog
-        condition:
-          type: "vrl"
-          source: 'includes(["worker"], .component)'
-
-      transform_task_log:
-        type: remap
-        inputs:
-          - filter_worker_logs
-          - filter_scheduler_logs
-        source: |-
-          # Parse Syslog input. The "!" means that the script should abort on error.
-          . = parse_json!(.message)
-          .@timestamp = parse_timestamp(.timestamp, "%Y-%m-%dT%H:%M:%S%Z") ?? now()
-          .check_log_id = exists(.log_id)
-          if .check_log_id != true {
```

Comment:
not sure if we need these params long term but they haven't been being applied thus far

Reply:
We added this check due to missing log id fields in some of the worker logs 
https://app.zenhub.com/workspaces/team-platform-apps-61d5f7ef8de56e0014840a75/issues/astronomer/issues/4648